server/server.go

```diff
@@ -669,6 +674,7 @@ func (s *Server) Kill(connectionID uint64, query bool) {
 	defer s.rwlock.RUnlock()
 	conn, ok := s.clients[connectionID]
 	if !ok {
+		s.dom.KillSysProcess(connectionID)
```

Comment:
Is it possible that some connection ID is the same as `reservedConnAnalyze`? In the case we just kill the connection rather than auto analyze.

Reply:
So I add `reservedLocalConns`, and connectionID will auto-increment from `reservedLocalConns`